akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala

```diff
@@ -315,7 +315,7 @@ class EventStreamSpec extends AkkaSpec(EventStreamSpec.config) {
       }
     }
 
-    "unsubscribe the actor, when it subscribes already in terminated state" in {
+    "unsubscribe the actor, when it subscribes already in terminated state" taggedAs GHExcludeTest in {
```

Comment:
Maybe we should link exclusions to tickets? In this case https://github.com/akka/akka/issues/30460